### PR TITLE
Display basic cursor and selection of remote users

### PR DESCRIPTION
**Pull request by dannelundqvist**

### CODE REVIEW

File: src/components/editor/editor.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react' // Necessary for esbuild
+import React, { PropsWithChildren } from 'react' // Necessary for esbuild
```
Comment (gularsson) [inline src/components/editor/editor.tsx:1]: PropsWithChildren is unused

File: src/components/editor/editor.tsx [hunk 2]
```diff
@@ -36,6 +36,8 @@ import { ContentToolbar } from './components/toolbar/content'
 import { InlineToolbar } from './components/toolbar/inline'
 import { withInsertBreak } from './with/insertBreak'
 import { withInsertHtml } from './with/insertHtml'
+import { RemoteCursorOverlay } from './components/presenceOverlay/remoteCursorOverlay'
```
Comment (gularsson) [inline src/components/editor/editor.tsx:39]: RemoteCursorOverlay is unused

**Review by gularsson — APPROVED** [review]
Looks great, as you said not much code. Great work :+1:

File: src/components/editor/editor.tsx (hunk 1 above)
Comment (dannelundqvist) [reply src/components/editor/editor.tsx]: 👍

File: src/components/editor/editor.tsx (hunk 2 above)
Comment (dannelundqvist) [reply src/components/editor/editor.tsx]: 👍

**Pull request merged**